### PR TITLE
Add PointCloudTutorial.md

### PR DESCRIPTION
**Pull request by SatjaSivcev**
This tutorial describes how to use Zivid SDK to work with Point Cloud data.

### CODE REVIEW

File: source/Applications/PointCloudTutorial.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+## Point Cloud
+
+### Get handle
+
+You can now get a handle to the point cloud data on the GPU ([go to source][point-cloud]).
+```csharp
+var pointCloud = frame.PointCloud;
+```
+The method ```Zivid.NET.Frame.PointCloud()``` does not perform any copying from GPU memory.
```
Comment (eskaur) [inline source/Applications/PointCloudTutorial.md:56]: It's not a method here, it's a "property" (note the lack of parenthesis in the snippet above).
Write something like: "Getting the property ```Zivid.NET.Frame.PointCloud``` does not perform any copying from GPU memory."

File: source/Applications/PointCloudTutorial.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Note: ```Zivid.NET.Camera.Capture()``` method returns as soon as the camera is done capturing. At that point the handle from ```Zivid.NET.Frame.PointCloud()``` is available instantly as well. However, the actual point cloud data becomes available only after the GPU is finished processing. This processing will occur on the GPU in the background, and any calls to data-copy functions (see section [Copy](#copy) below) will wait for this to finish before proceeding with the requested copy operation.
```
Comment (eskaur) [inline source/Applications/PointCloudTutorial.md:58]: Remove parenthesis from "Zivid.NET.Frame.PointCloud()".

Comment (eskaur) [reply source/Applications/PointCloudTutorial.md:58]: Also "data-copy functions" -> "data-copy methods".

File: source/Applications/PointCloudTutorial.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+### Copy
+
+You can now selectively copy data based on what is required. This is the complete list of output data formats and how to copy them from the GPU.
+
+
+|Return type|Functions for copying from GPU|Data per pixel|Total data copied|
```
Comment (eskaur) [inline source/Applications/PointCloudTutorial.md:65]: "Functions" -> "Methods"

File: source/Applications/PointCloudTutorial.md (hunk 2 above)
Comment (SatjaSivcev) [reply source/Applications/PointCloudTutorial.md:58]: image
Which ""?

Comment (SatjaSivcev) [reply source/Applications/PointCloudTutorial.md]: nevermind, done.

**Comment by SatjaSivcev** [comment]
Ready for the next round :)

**Comment by SatjaSivcev** [comment]
Thanks!

**Pull request merged**